commit e43c8918d574a88ccb2e0150c5ca396c9818d390
Author: Yang Bo <yangbo@deepin.com>
Date:   Thu Jun 8 10:36:51 2017 +0800

    Default to disable intel_pstate, use intel_pstate=enable to turn it on.
    
    Signed-off-by: Yang Bo <yangbo@deepin.com>

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index 283491f74..d2bdf8399 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -2450,7 +2450,7 @@ static int intel_pstate_update_status(const char *buf, size_t size)
 	return -EINVAL;
 }
 
-static int no_load __initdata;
+static int no_load __initdata = 1;
 static int no_hwp __initdata;
 static int hwp_only __initdata;
 static unsigned int force_load __initdata;
@@ -2702,8 +2702,8 @@ static int __init intel_pstate_setup(char *str)
 	if (!str)
 		return -EINVAL;
 
-	if (!strcmp(str, "disable")) {
-		no_load = 1;
+	if (!strcmp(str, "enable")) {
+		no_load = 0;
 	} else if (!strcmp(str, "passive")) {
 		pr_info("Passive mode enabled\n");
 		intel_pstate_driver = &intel_cpufreq;
